[rocminfo] undefined reference to `hsaKmt*`

Tried to compile `rocminfo` and got the following error log:

```
[100%] Linking CXX executable rocminfo
/usr/bin/cmake -E cmake_link_script CMakeFiles/rocminfo.dir/link.txt --verbose=1
/usr/bin/c++ -march=x86-64 -mtune=generic -O2 -pipe -fno-plt -fexceptions         -Wp,-D_FORTIFY_SOURCE=2 -Wformat -Werror=format-security         -fstack-clash-protection -fcf-protection -Wp,-D_GLIBCXX_ASSERTIONS -O3 -DNDEBUG -Wl,-O1,--sort-common,--as-needed,-z,relro,-z,now CMakeFiles/rocminfo.dir/rocminfo.cc.o -o rocminfo  -Wl,-rpath,/opt/rocm/lib /opt/rocm/lib/libhsa-runtime64.so.1.5.0
/usr/bin/ld: /opt/rocm/lib/libhsa-runtime64.so.1.5.0: undefined reference to `hsaKmtRuntimeEnable'
/usr/bin/ld: /opt/rocm/lib/libhsa-runtime64.so.1.5.0: undefined reference to `hsaKmtAcquireSystemProperties'
/usr/bin/ld: /opt/rocm/lib/libhsa-runtime64.so.1.5.0: undefined reference to `hsaKmtSetQueueCUMask'
/usr/bin/ld: /opt/rocm/lib/libhsa-runtime64.so.1.5.0: undefined reference to `hsaKmtGetNodeIoLinkProperties'
/usr/bin/ld: /opt/rocm/lib/libhsa-runtime64.so.1.5.0: undefined reference to `hsaKmtDestroyEvent'
/usr/bin/ld: /opt/rocm/lib/libhsa-runtime64.so.1.5.0: undefined reference to `hsaKmtMapMemoryToGPU'
/usr/bin/ld: /opt/rocm/lib/libhsa-runtime64.so.1.5.0: undefined reference to `hsaKmtQueryPointerInfo'
/usr/bin/ld: /opt/rocm/lib/libhsa-runtime64.so.1.5.0: undefined reference to `hsaKmtSetTrapHandler'
/usr/bin/ld: /opt/rocm/lib/libhsa-runtime64.so.1.5.0: undefined reference to `hsaKmtAllocQueueGWS'
/usr/bin/ld: /opt/rocm/lib/libhsa-runtime64.so.1.5.0: undefined reference to `hsaKmtReleaseSystemProperties'
/usr/bin/ld: /opt/rocm/lib/libhsa-runtime64.so.1.5.0: undefined reference to `hsaKmtWaitOnEvent'
/usr/bin/ld: /opt/rocm/lib/libhsa-runtime64.so.1.5.0: undefined reference to `hsaKmtCloseKFD'
/usr/bin/ld: /opt/rocm/lib/libhsa-runtime64.so.1.5.0: undefined reference to `hsaKmtUnmapMemoryToGPU'
/usr/bin/ld: /opt/rocm/lib/libhsa-runtime64.so.1.5.0: undefined reference to `hsaKmtShareMemory'
/usr/bin/ld: /opt/rocm/lib/libhsa-runtime64.so.1.5.0: undefined reference to `hsaKmtRegisterGraphicsHandleToNodes'
/usr/bin/ld: /opt/rocm/lib/libhsa-runtime64.so.1.5.0: undefined reference to `hsaKmtSetXNACKMode'
/usr/bin/ld: /opt/rocm/lib/libhsa-runtime64.so.1.5.0: undefined reference to `hsaKmtSetMemoryUserData'
/usr/bin/ld: /opt/rocm/lib/libhsa-runtime64.so.1.5.0: undefined reference to `hsaKmtSetMemoryPolicy'
/usr/bin/ld: /opt/rocm/lib/libhsa-runtime64.so.1.5.0: undefined reference to `hsaKmtGetNodeCacheProperties'
/usr/bin/ld: /opt/rocm/lib/libhsa-runtime64.so.1.5.0: undefined reference to `hsaKmtGetXNACKMode'
/usr/bin/ld: /opt/rocm/lib/libhsa-runtime64.so.1.5.0: undefined reference to `hsaKmtSetEvent'
/usr/bin/ld: /opt/rocm/lib/libhsa-runtime64.so.1.5.0: undefined reference to `hsaKmtUpdateQueue'
/usr/bin/ld: /opt/rocm/lib/libhsa-runtime64.so.1.5.0: undefined reference to `hsaKmtCreateEvent'
/usr/bin/ld: /opt/rocm/lib/libhsa-runtime64.so.1.5.0: undefined reference to `hsaKmtRegisterSharedHandleToNodes'
/usr/bin/ld: /opt/rocm/lib/libhsa-runtime64.so.1.5.0: undefined reference to `hsaKmtRegisterMemoryWithFlags'
/usr/bin/ld: /opt/rocm/lib/libhsa-runtime64.so.1.5.0: undefined reference to `hsaKmtGetTileConfig'
/usr/bin/ld: /opt/rocm/lib/libhsa-runtime64.so.1.5.0: undefined reference to `hsaKmtSVMGetAttr'
/usr/bin/ld: /opt/rocm/lib/libhsa-runtime64.so.1.5.0: undefined reference to `hsaKmtOpenKFD'
/usr/bin/ld: /opt/rocm/lib/libhsa-runtime64.so.1.5.0: undefined reference to `hsaKmtGetVersion'
/usr/bin/ld: /opt/rocm/lib/libhsa-runtime64.so.1.5.0: undefined reference to `hsaKmtFreeMemory'
/usr/bin/ld: /opt/rocm/lib/libhsa-runtime64.so.1.5.0: undefined reference to `hsaKmtGetNodeMemoryProperties'
/usr/bin/ld: /opt/rocm/lib/libhsa-runtime64.so.1.5.0: undefined reference to `hsaKmtGetClockCounters'
/usr/bin/ld: /opt/rocm/lib/libhsa-runtime64.so.1.5.0: undefined reference to `hsaKmtRegisterSharedHandle'
/usr/bin/ld: /opt/rocm/lib/libhsa-runtime64.so.1.5.0: undefined reference to `hsaKmtAllocMemory'
/usr/bin/ld: /opt/rocm/lib/libhsa-runtime64.so.1.5.0: undefined reference to `hsaKmtSVMSetAttr'
/usr/bin/ld: /opt/rocm/lib/libhsa-runtime64.so.1.5.0: undefined reference to `hsaKmtRuntimeDisable'
/usr/bin/ld: /opt/rocm/lib/libhsa-runtime64.so.1.5.0: undefined reference to `hsaKmtMapMemoryToGPUNodes'
/usr/bin/ld: /opt/rocm/lib/libhsa-runtime64.so.1.5.0: undefined reference to `hsaKmtGetNodeProperties'
/usr/bin/ld: /opt/rocm/lib/libhsa-runtime64.so.1.5.0: undefined reference to `hsaKmtDestroyQueue'
/usr/bin/ld: /opt/rocm/lib/libhsa-runtime64.so.1.5.0: undefined reference to `hsaKmtCreateQueue'
/usr/bin/ld: /opt/rocm/lib/libhsa-runtime64.so.1.5.0: undefined reference to `hsaKmtWaitOnMultipleEvents'
/usr/bin/ld: /opt/rocm/lib/libhsa-runtime64.so.1.5.0: undefined reference to `hsaKmtDeregisterMemory'
collect2: error: ld returned 1 exit status
```

I make more progress when linking rocminfo with hsa-amd-aqlprofile directly, but there are still some error messages of missing references:
```
/usr/bin/c++ -march=x86-64 -mtune=generic -O2 -pipe -fno-plt -fexceptions         -Wp,-D_FORTIFY_SOURCE=2 -Wformat -Werror=format-security         -fstack-clash-protection -fcf-protection -Wp,-D_GLIBCXX_ASSERTIONS -flto -O3 -DNDEBUG -Wl,-O1,--sort-common,--as-needed,-z,relro,-z,now -flto CMakeFiles/rocminfo.dir/rocminfo.cc.o -o rocminfo  -Wl,-rpath,/opt/rocm/lib /opt/rocm/lib/libhsa-runtime64.so.1.5.0 /opt/rocm/lib/libhsa-amd-aqlprofile64.so
/usr/bin/ld: /opt/rocm/lib/libhsa-runtime64.so.1.5.0: undefined reference to `hsaKmtSetXNACKMode'
/usr/bin/ld: /opt/rocm/lib/libhsa-runtime64.so.1.5.0: undefined reference to `hsaKmtGetXNACKMode'
/usr/bin/ld: /opt/rocm/lib/libhsa-runtime64.so.1.5.0: undefined reference to `hsaKmtSVMGetAttr'
/usr/bin/ld: /opt/rocm/lib/libhsa-runtime64.so.1.5.0: undefined reference to `hsaKmtGetClockCounters'
/usr/bin/ld: /opt/rocm/lib/libhsa-runtime64.so.1.5.0: undefined reference to `hsaKmtSVMSetAttr'
collect2: error: ld returned 1 exit status
```

--- a/hsakmt-roct/.SRCINFO
+++ b/hsakmt-roct/.SRCINFO
@@ -1,7 +1,7 @@
 pkgbase = hsakmt-roct
 	pkgdesc = Radeon Open Compute Thunk Interface
 	pkgver = 5.0.2
-	pkgrel = 1
+	pkgrel = 2
 	url = https://rocmdocs.amd.com/en/latest/Installation_Guide/ROCt.html
 	arch = x86_64
 	license = MIT
@@ -11,6 +11,7 @@ pkgbase = hsakmt-roct
 	depends = libdrm
 	provides = roct-thunk-interface=5.0.2
 	replaces = roct-thunk-interface
+	options = !lto
 	source = hsakmt-roct-5.0.2.tar.gz::https://github.com/RadeonOpenCompute/ROCT-Thunk-Interface/archive/rocm-5.0.2.tar.gz
 	sha256sums = f2a27ac18aada1dc0dba6455beb7dd7d88a4457c1917024ea372fecb03356e97
 

--- a/hsakmt-roct/PKGBUILD
+++ b/hsakmt-roct/PKGBUILD
@@ -6,7 +6,7 @@
 
 pkgname=hsakmt-roct
 pkgver=5.0.2
-pkgrel=1
+pkgrel=2
 pkgdesc='Radeon Open Compute Thunk Interface'
 arch=('x86_64')
 url='https://rocmdocs.amd.com/en/latest/Installation_Guide/ROCt.html'
@@ -18,6 +18,7 @@ replaces=('roct-thunk-interface')
 _git='https://github.com/RadeonOpenCompute/ROCT-Thunk-Interface'
 source=("${pkgname}-${pkgver}.tar.gz::$_git/archive/rocm-$pkgver.tar.gz")
 sha256sums=('f2a27ac18aada1dc0dba6455beb7dd7d88a4457c1917024ea372fecb03356e97')
+options=(!lto)
 _dirname="$(basename "$_git")-$(basename "${source[0]}" .tar.gz)"
 
 build() {

